packages/core/src/customer/strategies/stripe-upe/stripeupe-customer-initialize-options.ts

```diff
@@ -0,0 +1,26 @@
+export default interface StripeUPECustomerInitializeOptions {
+    /**
+     * The ID of a container which the stripe iframe should be inserted.
+     */
+    container: string;
+
+    /**
+     * A callback that gets called whenever the Stripe Link Authentication Element's value changes.
+     * @param authenticated - if the email is authenticated on Stripe.
+     * @param email - The new value of the email.
+     */
+    onEmailChange(authenticated: boolean ,email: string): void;
+
+    /**
+     * A callback that gets called when Stripe Link Authentication Element is Loaded.
+     */
+    isLoading(mounted: boolean): void;
+
+    /**
+     * get styles from store theme
+     */
+    getStyles(): {
+        [key: string]: string | undefined;
```

Comment:
Does this need to be undefined, for example can there be a style like 
```
color: undefined;
```
?

Reply:
On JS added validation to send undefined just in case BC styles didn't found in the dom
(JS code:
<img width="1155" alt="image" src="https://user-images.githubusercontent.com/42154828/188016804-470945c2-1389-4ea9-9147-e8a58af968be.png">

)
, as I wrote it on the SDK side (Typescript) seems wrong, sorry for that.

I think the correct approach would be    
```
 getStyles?(): {
    [key: string]: string;
 };
```

but let me know if It's ok

FYI: I added this change on the commit called `feat(payment): STRIPE-52 animesh & peng a proposal to solve the feedback`

Reply:
Same think this is much better :D